buildSrc/src/main/groovy/com/uber/okbuck/core/model/Scope.groovy

```diff
@@ -34,6 +40,7 @@ class Scope {
           DependencyCache depCache = OkBuckGradlePlugin.depCache) {
 
         this.project = project
+        this.rootPath = Paths.get(project.gradle.rootProject.projectDir.absolutePath)
```

Comment:
why is this a field? We can just do `project.rootDir` to get the root project dir inline